### PR TITLE
test(authentication app): Write unit tests for the current functionality

### PR DESCRIPTION
**Pull request by collinewait**
**What does this PR do?** 
   - It tests the current functionality and integrates with Travis CI.

**Description of Tasks to be completed?**
- Testing the current model and views in authentication app and adding a travis badge to show the build status.

**How should this be manually tested?**
- After cloning the repo, cd into the directory. 
- Run ```pip install -r requirements.txt```
- and finally Run ```python python manage.py test```

**What are the relevant pivotal tracker stories?**

- [#159951990](https://www.pivotaltracker.com/story/show/159951990)
- [#159951996](https://www.pivotaltracker.com/story/show/159951996)

### CODE REVIEW

**Comment by collinewait** [comment]
@DrKimpatrick  point noted. I am to improve that

**Review by reiosantos — COMMENTED** [review]
tests are failing to run. This is because you have not rebased onto develop. please do so.

File: authors/apps/authentication/tests.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+from django.test import TestCase
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from authors.apps.authentication.models import User
+
+
+class BaseTest():
+    """This class provides t"""
```
Comment (collinewait) [inline authors/apps/authentication/tests.py]: I am to write the complete class doc string

**Comment by amgoodwhoami** [comment]
@collinewait , your pivotal tracker numbers don't lint

**Comment by collinewait** [comment]
@huxaiphaer  I have linked them

**Pull request merged**